modules/gapi/src/backends/ov/govbackend.cpp

```diff
@@ -971,6 +1075,12 @@ struct Infer: public cv::detail::KernelTag {
         GAPI_Assert(uu.params.input_names.size() == in_metas.size()
                     && "Known input layers count doesn't match input meta count");
 
+        const auto &input_name = uu.params.input_names.at(0);
+        const auto &tensor_shape =
```

Comment:
Is `tensor_shape` is used somewhere?

Reply:
No, it's not. Removed, thanks! https://github.com/opencv/opencv/pull/24938/commits/833bd890ef003a7c161f3a91c61b9c8d81df0e55